openapi/swagger.yaml

```diff
@@ -13,27 +13,27 @@ servers:
 security:
   - ApiKey: []
 tags:
-  - name: Creditor Istitutions
-    description: Everything about Creditor Istitution
+  - name: Creditor Institutions
```

Comment:
Great , i think we can close this [PR](https://github.com/pagopa/pagopa-api/pull/203) and reference  the current one swagger file into design review document.
Are you agree @corvettafabio ?